etc/cn-minimal-test.toml

```diff
@@ -0,0 +1,125 @@
+# service node type, [DN|CN|LOG]
+service-type = "CN"
+
+[log]
+level = "debug"
+format = "json"
+max-size = 512
+
+[cn.frontend]
```

Comment:
The default configuration needs to be removed from the configuration to keep it simple, and the default value is set inside the code

Reply:
> The default configuration needs to be removed from the configuration to keep it simple, and the default value is set inside the code

I will remove the parameters using the default value

Reply:
I recommend comment out optional configuration items, so that users can easily inspect all the configuration items and default values in one place while we still keep the minimal configuration simple.

Reply:
How do you like? @zhangxu19830126 @daviszhen

Reply:
Also, we don't need a full test toml in etc if we do so. (maybe we still need a full one in bvt to all the test configuration items are working)

Reply:
We may need the rule to classify those configured parameters into:

- parameters can be modified by users(need to expose to users by default)
- parameters can but are not recommended for modification
- parameters cannot be modified by the user

Reply:
btw, better to remove all these parameters used by the internal modules, like `processLimitationSize`, `portOfRpcServerInComputationEngine` and etc. I don't think users need to know them at all.

Reply:
> How do you like? @zhangxu19830126 @daviszhen
I keep all the parameters of the frontend with the default values and remove them from the cn-minimal

Reply:
> Also, we don't need a full test toml in etc if we do so. (maybe we still need a full one in bvt to verify all the configuration items are working)

I remove it.

Reply:
> btw, better to remove all these parameters used by the internal modules, like `processLimitationSize`, `portOfRpcServerInComputationEngine` and etc. I don't think users need to know them at all.

I remove them.